regulations/static/regulations/css/less/module/comment.less

```diff
@@ -333,8 +333,9 @@ Write Mode
   .comment-index-items {
     margin: 0 0 15px 0;
 
-    & > h3 {
+    & > h4 {
       margin: 0.6em 0 0 0;
+      text-transform: none;
```

Comment:
Looks like this rule specifically applies to the text "Your Comment", which doesn't have a text-transform we need to unset? Similar to the below.